scripts/README.md

```diff
@@ -0,0 +1,3 @@
+The scripts in this directory are used to populate prod like environments in the following fashion:
+
+- `cat $CSV_DOWNLOADED_FROM_BT | python process_sender_names.py`  - emits an insert statement that has a list of senderids to populate a table that lists those that are protected from being set. Admin will also raise a zendesk ticket if someone tries to set their senderid to one of those protected. It filters out all government and healthcare sender ids as we want those to be able to be set by our users.
```

Comment:
```suggestion
- `cat $CSV_DOWNLOADED_FROM_BT | python process_sender_names.py`  - this script takes a list of protected sender names that we get from BT and uses that list to populate a db table. We then use that table to check if a sender name our user wants to set is protected. If it's one of the protected ones, we don't set it, and we also raise an alert in a form of a Zendesk ticket (code for that is in admin repo). The script filters out all government and healthcare sender names as we want our service users to be able to use them.
```

Reply:
I edited the script description for ease of reading, I hope it's ok.

Also are we going to run this script manually or programmatically? It's not mentioned.